lib/msf/util/ruby_deserialization.rb

```diff
@@ -0,0 +1,44 @@
+# -*- coding: binary -*-
+
+# Ruby deserialization Utility
+module Msf
+  module Util
+    # Ruby deserialization class
+    class RubyDeserialization
+      # That could be in the future a list of payloads used to exploit the Ruby deserialization vulnerability.
+      # TODO: Add more payloads
+      # TDOO: Create a json file with the payloads?
+      PAYLOADS = {
+        'Universal' => {
```

Comment:
I'm not sure if 'universal' is a good long-term name here; i.e. it's already patched in Ruby 3.0.2, and there's already existing 'universal' payloads that wouldn't work for this exploit